cmd/root.go

```diff
@@ -70,12 +73,12 @@ func init() {
 	// Here you will define your flags and configuration settings.
 	// Cobra supports persistent flags, which, if defined here,
 	// will be global for your application.
-
 	rootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is $HOME/.fsoc.yaml)")
 	rootCmd.PersistentFlags().StringVar(&cfgProfile, "profile", "", "access profile (default is current or \"default\")")
 	rootCmd.PersistentFlags().StringVarP(&outputFormat, "output", "o", "auto", "output format (auto, table, detail, json, yaml)")
 	rootCmd.PersistentFlags().String("fields", "", "perform specified fields transform/extract JQ expression")
 	rootCmd.PersistentFlags().BoolP("verbose", "v", false, "Enable detailed output")
+	rootCmd.PersistentFlags().String("log-loc", "/tmp", "determines the location of the fsoc log file")
```

Comment:
It will be better to allow specifying the file rather than the containing directory. This will allow using `/dev/null` (to disable the log) or temp (or numbered) filename to preserve past logs. Let's use `--log` as a flag and having it be the complete path (defaults to `path.Join(os.TempDir, "fsoc.log")`, described in the help as "fsoc.log in the OS temp directory")